RELEASE_NOTES.md

```diff
@@ -1,44 +1,16 @@
-# ChefDK 0.12 Release notes
+# ChefDK 0.14 Release notes
 
-## New / Updated Packages
+## New tools added to the ChefDK
 
-This version of ChefDK includes a number of key updates to dependent packages:
+We're adding some additional tools to the ChefDK to make it easier for cookbook developers using Delivery. The tools are:
 
-* chef - updated to 12.8.1
-* ohai - updated to 8.12.1
-* inspec - updated to 0.15.0
-* kitchen-inspec - updated to 0.12.13
-* berkshelf - updated to 4.3.0
-* test-kitchen - updated to 1.6.0
-* knife-windows - updated to 1.4.0
-* foodcritic - updated, 6.0.1
-* openssl - updated to 1.0.1s
-* chefspec - updated 4.6.0
-* winrm-fs - new dependency 0.3.2
+* Delivery CLI
+* Git
+* Push Jobs client
+* The following gems:
+  * [Chef Sugar](https://github.com/sethvargo/chef-sugar)
+  * [Knife Supermarket](https://github.com/chef/knife-supermarket)
+  * [Artifactory](https://github.com/chef/artifactory-client)
+  * [Mixlib Versioning](https://github.com/chef/mixlib-versioning)
 
-## Chef 12.8.1
-
-Please see the Release Notes (https://github.com/chef/chef/blob/master/RELEASE_NOTES.md)
-for information on the key changes in Chef 12.8.1.
-
-## Knife-windows 1.4.0
-
-* Users can specify the architecture they want to install on the target system during knife bootstrap windows.
-* `client.rb` can enforce FIPS mode.
-* A SSL fingerprint can be specified when using self signed certificates.
-* Support for Extended Protection for Authentication (aka Chennel Binding) over SSL.
-
-See the knife-windows Changelog for more details (https://github.com/chef/knife-windows/blob/master/CHANGELOG.md)
-
-## Berkshelf 4.3.0
-
-Supports downloading universe from chef servers.
-
-See the berkshelf Changelog for more details (https://github.com/berkshelf/berkshelf/blob/master/CHANGELOG.md)
-
-## Test-Kitchen 1.6.0
-
-* Fixes WinRM authorization errors using kitchen-inspec
-* Adds a `chef_apply` provisioner
-
-Lots of other bug fixes and improvements can be found in the test-kitchen changelog (https://github.com/test-kitchen/test-kitchen/blob/master/CHANGELOG.md)
+Because existing users may have a `git` client setup we do not put our bundled `git` executable on the path for most operating systems. Windows users commonly request that we include `git` so we _are_ adding our bundled executable to the path. There is an install-time flag you can use to disable this. If you run `chef shell-init` it will add our bundled `git` executable to your path.
```

Comment:
@oferrigni We don't know yet because I moved that work into an external card. I'll update that card to add an acceptance criteria for updating this README and any install wizard docs